src/dispatch/static/dispatch/src/case/type/NewEditSheet.vue

```diff
@@ -221,7 +222,10 @@ export default {
   created() {
     if (this.$route.query.project) {
       this.project = { name: this.$route.query.project }
```

Comment:
Probably makes sense to take out this initial assignment then, since the subsequent assignment is not optional.